packages/common/src/utils/urlFormatting.ts

```diff
@@ -117,6 +117,10 @@ export const mediaURL = ({
   );
 };
 
+export const singleMediaURL = (id: string, title?: string) => {
```

Comment:
@ChristiaanScheermeijer I think it makes sense to rewrite `mediaURL` function to accept id and title instead of the whole media item. Though I also see that it is used in many places, so I would prefer to do it in a separate PR.

Reply:
Sounds good!

Reply:
Alternatively, for now you could do this as well to prevent refactoring:

```tsx
export const mediaURL = ({
  media,
  id,
  title,
  playlistId,
  play = false,
  episodeId,
}: ParamsWithMedia | ParamsWithId) => {
  const mediaId = media ? media.mediaid : id;
  const mediaTitle = media ? media.title : title;

  // ...
}
```

Reply:
I created a separate ticket: https://jwplayer.atlassian.net/browse/OWA-89 🚀
I don't know if Christiaan can see it in Jira, but we'll tackle this in a separate PR 🙂